pkg/deployment/deployment.go

```diff
@@ -215,6 +215,11 @@ func (d *Deployment) run() {
 			d.CreateEvent(k8sutil.NewErrorEvent("Failed to create pods", err, d.GetAPIObject()))
 		}
 
+		// Create PDBs
```

Comment:
Probably helpful to spell our PodDisruptionBudgets completely in this comment. PDB is not that well known